Makefile.include

```diff
@@ -193,6 +195,15 @@ ifeq ($(DEBUG),1)
   CFLAGS += $(CFLAGS_DEBUG)
 endif
 
+# Default to not use static analysis.
+ANALYZE ?= 0
+ifeq ($(ANALYZE),1)
+  # Define something corresponding to __clang_analyzer__ for GCC as well.
+  CFLAGS += -fanalyzer -D__gcc_analyzer__ $(CFLAGS_FANALYZER)
+  # FIXME: poor -fanalyzer results with GCC 13 according to documentation.
```

Comment:
You mean gcc 10 here?

Reply:
I mean GCC 13, but I'm sure the statement is true for GCC 10 as well.

Reply:
Ah, sorry I found what you are referring to now